.vitepress/config.mts

```diff
@@ -90,6 +91,11 @@ export default defineConfig({
 
   },
 
+  rewrites: {
```

Comment:
@FabianWilms I trid https://vitepress.dev/guide/routing#route-rewrites but this does not work

Reply:
according to the docs I think you have to do it like this:

```
    'overview.md': 'software/index.md',
    'de/overview.md': 'de/software/index.md',
```

Reply:
@FabianWilms you are right! thx

but did not work either

Reply:
> but did not work either

what exactly did not work? After a build there should be an index.html-File under the `software` directory with the rendered contents of `overview.md`.

It won't automatically redirect from `opensource.muenchen.de/overview.html` to `opensource.muenchen.de/software/index.html`

Reply:
fixed